### PR TITLE
Update CardChecker.java

### PR DESCRIPTION
**Pull request by TalSerenVia**
I made CR for this. it's in comments

### CODE REVIEW

File: src/main/java/cards/CardChecker.java [hunk 1]
```diff
@@ -32,6 +35,7 @@ public CardCombination getCombination(List<CardModel> cardModels) {
         Set<Map.Entry<CardModel, Long>> groupedCardModelEntries = groupedCardModel.entrySet();
 
         switch (groupedCardModelEntries.size()) {
+            // CR: Please move to constants that describe what each case represents
```
Comment (IamEgor) [inline src/main/java/cards/CardChecker.java:38]: Answer: Its just a quantity of unique cards in set. There is no matter to create constants for this case

File: src/main/java/cards/CardChecker.java [hunk 2]
```diff
@@ -71,6 +77,8 @@ private boolean isStreet(List<CardModel> cardModels) {
         return isStreet;
     }
 
+    // CR: in getCombination() you are receiving a List and not a set?
+    // is there a reason for the difference?
```
Comment (IamEgor) [inline src/main/java/cards/CardChecker.java:81]: I'm receiving a List. Than it was grouped to Map [Rank -> quantity of card in list]. The was created to iterate over this map.

File: src/main/java/cards/CardChecker.java (hunk 1 above)
Comment (TalSerenVia) [reply src/main/java/cards/CardChecker.java:38]: In that case, THREE_UNIQUE_CARDS_IN_HAND could be a good name for the constant.
I understood your algorithm but sometimes when it becomes more complex and people add many things to it starts to get less and less readable.

File: src/main/java/cards/CardChecker.java (hunk 2 above)
Comment (TalSerenVia) [reply src/main/java/cards/CardChecker.java:81]: my question is if there is a reason that in getCombination() you are receiving a List, and in isFourOfKind() you are receiving a Set (of a Map).
Both list and set receive a single type (unlike map) and are iterated in the same way.
is there a value to receive a List in one place and Set in another?

File: src/main/java/cards/CardChecker.java (hunk 1 above)
Comment (IamEgor) [reply src/main/java/cards/CardChecker.java:38]: I get your point. Agree

File: src/main/java/cards/CardChecker.java (hunk 2 above)
Comment (IamEgor) [reply src/main/java/cards/CardChecker.java:81]: isFourOfKind() should receive a Map [Rank -> quantity of card in list]. A Set should be created, to iterate through it. Actually, three is no need  to create this Set outside of isFourOfKind() method scope but it should been created in any case for iteration process.

**Pull request merged**